src/javascript/binary/static/viewbalance/viewbalance.ui.js

```diff
@@ -0,0 +1,58 @@
+
+var ViewBalanceUI = (function(){
+    var colsName = ["loginid", "currency", "balance"];
+
+    function createEmptyBalanceTable(){
+        var header = [
+            Content.localize().textLoginID,
+            Content.localize().textCurrency,
+            Content.localize().textBalance
+        ];
+        var metadata = {
+            cols: colsName,
+            id: "bal-table"
+        };
+        var data = [];
+        var $table = Table.createFlexTable(data, metadata, header);
+
+        return $table;
+    }
+
+    function createBalancePopup(){
+        var $popupDiv = $("<div></div>", {class: "popup-div", id: "balance-container"});
+        var $balTitle = $("<h1></h1>", {text: Content.localize().textBalances});
+        var $br = $("<br>");
+        var $table = createEmptyBalanceTable();
+
+        var $span = $("<span></span>", {class: "button"});
+        var $button = $("<button></button>", {
+            class: "button",
+            id: "close-balances",
+            text:Content.localize().textContinueTrading
+        });
+        $span.append($button);
+
+        $popupDiv.append($balTitle).append($br).append($table).append($span);
```

Comment:
@raunak-binary 

I was wrong, JQuery does use Document Fragment, when you passed in an array but not in loops, so not relevant in the code here

But I think if DocFragment helps because it delay reflow, a detached DOM (which is used here) should serve the same purpose, because reflow only happens when we append to the view

One of the answer here have similar opinion, what do you think?
http://stackoverflow.com/questions/14615484/does-jquery-use-create-document-fragment-inside-each-loops

I am reluctant to change as documentFragment API does not accept JQuery object out of the box, haha